bundles/core/org.openhab.core.library/src/main/java/org/openhab/core/library/types/PointType.java

```diff
@@ -113,6 +118,24 @@ public DecimalType getGravity() {
 	}
 
 	/**
+	 * Return the distance in meters from otherPoint, ignoring altitude. This algorithm also
```

Comment:
cool, I haven't used that one before. Might be useful to update the javadoc with this info.